src/resources/js/frontend-ticket-form.js

```diff
@@ -30,9 +30,15 @@ var tribe_tickets_ticket_form = {};
 			my.normal_stock_quantity_changed( $this, ticket_id );
 		}
 
+		var $form = $this.closest( 'form' );
+
+		// Only disable / enable if is a Tribe Commerce Paypal form.
+		if ( ! $form.hasClass('tribe-tickets-tpp') ) {
```

Comment:
not a blocker but remember to have spaces after the opening `(` and before the closing `)`:
```
$form.hasClass( 'tribe-tickets-tpp' )
```